admin/src/js/services/validate-form.js

```diff
@@ -0,0 +1,29 @@
+angular.module('inboxServices').factory('ValidateForm',
```

Comment:
Please add a couple of unit tests for this service.

Reply:
It only consumes an external service, but it does not do anything clever to test.